tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -280,6 +374,8 @@ use for building. For the purpose of this example, just use
 we've prepared. We'll cover a more customizable, detailed example on
 how to use these build files in a subsequent tutorial.
 
+.. note:: This ``SConstruct`` file was written to be used with the latest Godot-cpp master, you may need to make small changes using it with older versions or refer to the ``SConstruct`` file in the Godot 3.0 documentation.
```

Comment:
Should be `godot-cpp` without capitalization I guess?

Reply:
Sure :)